Archive old YubiHSM2 documentation

bkanui opened the pull request:
Two pieces:
-- Copy of latest version of the developers.yubico.com/content/YubiHSM documentation copied to an archive folder /Archive_Old_Dev_Docs. 
   = Added a Readme to explain this process.
-- Updated all the documentation files in developers.yubico.com/content/YubiHSM
    = Removed content from each file
    = Added statement: This content is deprecated. See <link to docs.yubico.com> 
    = Added link to replacement YubiHSM 2 User Guide and link to specific topic within that guide.

FFincannon commented:
Hi @joostd , can I ask you to please merge this PR? Saqib asked us to move the HSM docs from developers.yubico.com, so we've done that, and these are the redirects. I am making sure that the recent updates to this old content are transferred to the new content on docs.yubico.com, updates such as PR#568

joostd commented on content/Software_Projects/Yubico_OTP/Yubico_OTP_Integration_Plug-ins/index.adoc:
Seems like there is still an unresolved merge conflict in this and other files

joostd commented on content/Software_Projects/Yubico_OTP/YubiCloud_Validation_Servers/index.adoc:
see other comment about unresolved merge conflicts

joostd commented on content/Software_Projects/Yubico_OTP/YubiCloud_Connector_Libraries/index.adoc:
see other comment about unresolved merge conflicts

joostd commented on content/OTP/OTPs_Explained.adoc:
see other comment about unresolved merge conflicts

joostd commented on content/OTP/Guides/Self-hosted_OTP_validation.adoc:
see other comment about unresolved merge conflicts

joostd commented on content/OTP/Guides/Migrating_to_python-pyhsm.adoc:
see other comment about unresolved merge conflicts

joostd commented on content/Developer_Program/Guides/Securing_Web_Services_with_OTP.adoc:
see other comment about unresolved merge conflicts

joostd commented on content/Developer_Program/Guides/Touch_triggered_OTP.adoc:
see other comment about unresolved merge conflicts

joostd reviewed (commented):
There are some merge conflicts that still need to be resolved. See individual comments on affected files (not sure if I commented them all, please check).
Otherwise this PR lgtm!

bkanui commented:
Hello @joostd 
I apologize for taking so long to resolve your conflicts. I thought I had already cleared them. 
Please merge this PR, or let me know if there are other issues. I'll attend to them right away!
Thanks,

joostd commented:
For some reason I don't seem to be authorized to merge. Assigning to @elukewalker

joostd commented:
There still seem to be some merge conflicts left to fix:

```
$ git grep ">>>>>>>"
content/Developer_Program/Guides/Securing_Web_Services_with_OTP.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
content/Developer_Program/Guides/Touch_triggered_OTP.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
content/OTP/Guides/Migrating_to_python-pyhsm.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
content/OTP/Guides/Self-hosted_OTP_validation.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
content/OTP/OTPs_Explained.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
content/Software_Projects/Yubico_OTP/YubiCloud_Connector_Libraries/index.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
content/Software_Projects/Yubico_OTP/YubiCloud_Validation_Servers/index.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
content/Software_Projects/Yubico_OTP/Yubico_OTP_Integration_Plug-ins/index.adoc:>>>>>>> 456647ee9dcca05aafc4d68935558d154423db61
```

bkanui commented:
commit b218db6, merge intending to resolve conflicts.  message '-a' was a typo.

bkanui commented:
@joostd  Could you please check if there are more conflicts? I went through all the files listed as changed and updated then committed, see commit b218db6. But when I view 'all files' it still shows brown/+ for conflict. 

When I check my master branch git responds with branch is up to date with origin/master. 
When I check my bk-hsm2-oldbook  git respond nothing to commit, working tree clean. 
Both seem to be inline with doing a 'simple' update, but it is not working. 

I did a test and updated plugins.adoc from pr to git. Still shows conflict. 

Any suggestions for how to resolve these? Should I close this PR, update local master, create a new branch, apply changes (I saved off a copy), pull commit and push, make a new PR?

joostd commented:
With your latest commits, I no longer see any conflicts. Fine to merge as far as I am concerned!

FFincannon commented:
I know, @elukewalker - I see that too - I just don't understand how those conflicts could be resolved to Joost's satisfaction, and then REAPPEAR !
